data-pipeline/README.md

```diff
@@ -58,6 +58,11 @@ The ClinVar pipelines must be run on clusters with an appropriate version of VEP
   ./deployctl dataproc-cluster start <cluster-name>
   ```
 
+  NOTE: Starting with `gnomad version 4.1`, it is highly recommended to specify a
+  custom OS image that was built for that gnomAD version. To see these images, run
+  `gcloud compute images list --filter="family=dataproc-custom-image"`. You can then
```

Comment:
Running this tells me 0 images were found:

```
❯ gcloud compute images list --filter="family=dataproc-custom-image"
Listed 0 items.
```

My gcloud config's project is set to `exac-gnomad`, maybe that's the reason?